src/Microsoft.DotNet.Build.Tasks.Feed/PushToBlobFeed.cs

```diff
@@ -35,14 +34,17 @@ public class PushToBlobFeed : MSBuild.Task
 
         public bool Overwrite { get; set; }
 
+        public bool IsSecure{ get; set; }
```

Comment:
Why do we need to do this differently? I would have expected to use the same URL with the SAS token embedded in the path for the publish as we do with restore.